packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -3250,7 +3250,7 @@ const ruleset = {
             message: "{{error}}",
             resolved: true,
             formats: [oas2],
-            given: "$[paths,'x-ms-paths'][?(!@property.endsWith('}'))].get",
+            given: "$[paths,'x-ms-paths'][?(!@property.endsWith('}') && !@property.endsWith('/default'))].get",
```

Comment:
>default [](http://example.com/codeflow?start=94&length=7)

There could be other terms that maybe used - not just default. You can get this in for now as the maority case is when the term 'default' is sed. Pl open another item to cover for other cases. A better approach could be to make this check for odd segmented GET paths